src/core/Akka.Cluster.Tests/ClusterGenerators.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using Akka.Actor;
+using Akka.Tests.Shared.Internals.Helpers;
+using FsCheck;
+
+namespace Akka.Cluster.Tests
+{
+    public class ClusterGenerators
```

Comment:
Could technically delete this class - used it for some FsCheck tests I deleted as they were no longer needed